docs/ingestion/index.md

```diff
@@ -309,8 +309,8 @@ Example ingestion spec for task type `index_parallel` (native batch):
       },
       "dimensionsSpec": {
         "dimensions": [
-          { "type": "string", "page" },
-          { "type": "string", "language" },
+          { "type": "string", "name": "page" },
```

Comment:
I think we could use "page" directly instead of an json object. See Line 448 and 472 of this file.

Reply:
lines 385-386, 663-664 also have this same mistake

Reply:
Thanks @clintropolis , I have fixed them all.